modules/nlu/src/backend/application/training-queue.ts

```diff
@@ -265,4 +258,17 @@ export class TrainingQueue {
     const modifiedOn = new Date()
     return { status, owner, modifiedOn, progress: progress ?? 0 }
   }
+
+  public teardown = async (): Promise<void[]> => {
```

Comment:
why moving this at the end ? not against, just curious

Reply:
While rewriting stuff I deleted it, I put it back at the end, I always put teardown functions at the end when I can